interpreter/cling/lib/Interpreter/ClingPragmas.cpp

```diff
@@ -60,6 +78,24 @@ namespace {
       Token& CurTok = const_cast<Token&>(P.getCurToken());
       CurTok.setKind(tok::semi);
 
+      return Literal;
+  }
+
+  class PHLoad: public PragmaHandler {
+    Interpreter& m_Interp;
+
+  public:
+    PHLoad(Interpreter& interp):
+      PragmaHandler("load"), m_Interp(interp) {}
+
+    void HandlePragma(Preprocessor &PP,
+                      PragmaIntroducerKind Introducer,
+                      Token &FirstToken) override {
+      // TODO: use Diagnostics!
+      std::string file;
+      file = HandlePragmaHelper(PP, Introducer, FirstToken, m_Interp,
+                                           "cling::PHLoad: expect '(' after #pragma cling load!\n", "pragma cling load");
+      Preprocessor::CleanupAndRestoreCacheRAII cleanupRAII(PP);
```

Comment:
Before going here, make sure that file is non-empty, else `llvm::errs() <<...` some informative error message that you cannot load unnamed files.